admin/client/EditorTextTab.tsx

```diff
@@ -32,8 +32,17 @@ export class EditorTextTab extends React.Component<{ editor: ChartEditor }> {
         }
     }
 
+    componentDidMount() {
+        if (!this.props.editor.chart.props.relatedQuestions) {
```

Comment:
Hm, I think this would still fail for the case when `relatedQuestions = []`?
Maybe worth having another check...

Reply:
Correct (even though you couldn't get an empty array at the time, but good for future devs). 

Either way, I went ahead and made this user driven, which prevents adding an empty object to the config as soon as the text tab is loaded (which didn't fill me with joy either :))

For reference, this paves the way for supporting multiple related questions (a potential future project).